pkgs/development/tools/ilspy/default.nix

```diff
@@ -0,0 +1,56 @@
+{ pkgs ? (import <nixpkgs> {})}:
```

Comment:
Instead of taking the whole package set as an input, use the individual packages as inputs:

```nix
{ stdenv, libX11, kerberos, curl, lttng-ust, gtk3, openssl, icu }:
```

Reply:
done in 2d064c120915267b65d41a1d9f3ddd51021db4da